packages/shared/src/util.ts

```diff
@@ -21,6 +21,20 @@ export function titleizeImageryName(name: string): string {
   );
 }
 
+/**
+ * Make a tile imagery title as imagery name
+ * @example
+ *  'Tasman rural 2018-19 0.3m' => 'tasman_rural_2018-19_0-3m'
+ */
+export function nameImageryTitle(title: string): string {
+  return title
+    .replace(/ /g, '_')
+    .replace('.', '-')
```

Comment:
this only does the first `.` might want to remove all `.` 's

Reply:
All fixed, thank you.